sublime_jedi/py_select.py

```diff
@@ -0,0 +1,44 @@
+import sublime, sublime_plugin
+from .utils import PythonCommandMixin, is_python_scope
+from .settings import get_plugin_settings
+
+
+class PySelectEventListener(sublime_plugin.EventListener):
+
+    def show_py(self, view) -> None:
+        '''
+        Show the Python Interpreter used by SublimeJEDI on statusbar
+        '''
+        if is_python_scope(view, view.sel()[0].begin()):
+            view.window().status_message('   Py: %s' %get_plugin_settings().get('python_interpreter'))

```

Comment:
I think we should use format where it is possible due to python3 is here :)

Reply:
I have changed % to .format().
I'm not familiar with github. This is my first pull request. So it took me a while to figure out what I needed to do